vulnerabilities/importer_yielder.py

```diff
@@ -61,16 +61,16 @@
             'debian_tracker_url': 'https://security-tracker.debian.org/tracker/data/json'
         },
     },
-#     {
-#         'name': 'safetydb',
-#         'license': 'cc-by-nc-4.0',
-#         'last_run': None,
-#         'data_source': 'SafetyDbDataSource',
-#         'data_source_cfg': {
-#             'url': 'https://raw.githubusercontent.com/pyupio/safety-db/master/data/insecure_full.json',  # nopep8
-#             'etags': {}
-#         },
-#     },
+    {
```

Comment:
We need to have a way to selectively enable some non-free resources like this one. As long as this is under a cc-by-nc  license we cannot integrate these data by default IMHO 
@jayfk ping?